src/core/openthread-core-default-config.h

```diff
@@ -102,7 +102,7 @@
  *
```

Comment:
How about adding a comment here to explain why using a multiple of `sizeof(void *)`?